packages/terra-clinical-result/src/ClinicalResult.jsx

```diff
@@ -47,12 +48,13 @@ const propTypes = {
    */
   isTruncated: PropTypes.bool,
   /**
-   *  If the Result value has not been authenticated and committed to patient chart.
-   */
+    * Add indicator if the result value has been modified to a different value from its
```

Comment:
Let's be consistent with the first change:

```suggestion
    * Adds indicator if the result value has been modified to a different value from its
```